.github/actions/spelling/patterns.txt

```diff
@@ -1,7 +1,112 @@
 # See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns
 
-# ignore long runs of a single character:
-\b([A-Za-z])\g{-1}{3,}\b
+# Automatically suggested patterns
```

Comment:
This section was more or less generated based on the `candidates` file above (with some manual tuning).